entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/transformer/TypeDeclarationsToClassWithMemberTransformer.kt

```diff
@@ -0,0 +1,44 @@
+package godot.entrygenerator.transformer
+
+import godot.entrygenerator.model.ClassWithMembers
+import org.jetbrains.kotlin.descriptors.ClassDescriptor
+import org.jetbrains.kotlin.descriptors.FunctionDescriptor
+import org.jetbrains.kotlin.descriptors.PropertyDescriptor
+
+object TypeDeclarationsToClassWithMemberTransformer {
+
+    fun transform(
```

Comment:
I would rather prefer a top level method, but that's a personal preference.

Reply:
changed it